airflow/example_dags/example_bash_decorator.py

```diff
@@ -33,7 +33,7 @@ def example_bash_decorator():
     def run_me(sleep_seconds: int, task_instance_key_str: str) -> str:
         return f"echo {task_instance_key_str} && sleep {sleep_seconds}"
 
-    run_me_loop = [run_me.override(task_id=f"runme_{i}")(sleep_seconds=i) for i in range(3)]
+    run_me_loop = [run_me.override(task_id=f"runme_{i}")(sleep_seconds=i) for i in range(5)]
```

Comment:
Did you indend to bump this permanently, or was that a leftover experiment?

Reply:
Oops. Leftover ..